services/base.spec.js

```diff
@@ -37,12 +37,12 @@ describe('BaseService', () => {
     });
 
     it('registers the service', () => {
-      assert(mockCamp.route.calledOnce);
-      assert.equal(mockCamp.route.getCall(0).args[0].toString(), expectedRouteRegex);
+      expect(mockCamp.route).to.have.been.calledOnce;
+      expect(mockCamp.route.getCall(0).args[0].toString()).to.equal(expectedRouteRegex.toString());
```

Comment:
Can we replace it with
```js
expect(mockCamp.route).to.have.been.calledWith(expectedRouteRegex);
```

Reply:
Unfortunately it fails, because we need the `.toString()`.

```nodeconsole
> /foo/ === /foo/
false
> /foo/ == /foo/
false
> /foo/.toString() === /foo/.toString()
true
```

Reply:
Ha, no, you're right.

Reply:
❤️ Chai